bootstrap.d/dev-qt.yml

```diff
@@ -0,0 +1,395 @@
+sources:
+  - name: qt6
+    subdir: 'ports'
+    # This may seem strange, but is actually correct
+    git: 'https://code.qt.io/qt/qt5.git'
+    tag: 'v6.3.0'
+    version: '6.3.0'
+    regenerate:
+      # Get the submodules that we'll need as a host tool
+      - args: ['./init-repository', '-f', '--module-subset', 'qtbase,qtsvg,qtimageformats,qtshadertools,qtlanguageserver,qtdeclarative,qtwayland,qtmultimedia']
+
+tools:
+  # We do a limited superbuild with only the needed modules of Qt6 to satisfy host tooling.
+  - name: host-qt6
+    from_source: qt6
+    configure:
+      - args:
+        - 'cmake'
+        - '-GNinja'
+        - '-DCMAKE_INSTALL_PREFIX=@PREFIX@'
+        - '-DBUILD_SHARED_LIBS=ON'
+        - '-DQT_BUILD_EXAMPLES=OFF'
+        - '-DQT_BUILD_TESTS=OFF'
+        - '@THIS_SOURCE_DIR@'
+    compile:
+      - args: ['cmake', '--build', '.', '--parallel', '@PARALLELISM@']
+    install:
+      - args: ['cmake', '--install', '.']
+
+packages:
+  - name: qtbase6
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: The Qt6 base libraries
+      description: This package provides the Qt6 base libraries, including QtCore, QtGui, QtWidgets and QtNetwork.
+      spdx: 'GPL-2.0-only GPL-3.0-only LGPL-3.0-only GFDL-1.3-only'
+      website: 'https://code.qt.io/cgit/qt/qtbase.git/'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
+      categories: ['dev-qt']
+    source:
+      subdir: 'ports'
+      git: 'https://code.qt.io/qt/qtbase.git'
+      tag: 'v6.3.0'
+      version: '6.3.0'
+    tools_required:
+      - system-gcc
+      - host-cmake
+      - host-qt6
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: x86_64-managarm
```

Comment:
This should be `triple: "@OPTION:arch-triple@"`

Reply:
Good catch, fixing.